crates/astria-proto/proto/astria/sequencer/v1alpha1/transaction.proto

```diff
@@ -28,6 +28,7 @@ message Action {
         TransferAction transfer_action = 1;
         SequenceAction sequence_action = 2;
         tendermint.abci.ValidatorUpdate validator_update_action = 3;
+        MintAction mint_action = 5;
```

Comment:
Why is this 5 and not 4?

Reply:
4 is used here: https://github.com/astriaorg/astria/pull/431

Reply:
so I just did this in anticipation haha